Source/DafnyDriver/DafnyDriver.cs

```diff
@@ -592,20 +592,25 @@ private record TargetPaths(string Directory, string Filename) {
 
       WriteFile(paths.Filename, targetProgramText, callToMain);
 
+      var relativeTarget = paths.RelativeFilename;
+      if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows)) {
+        // To satisfy the tests that assume a '/' instead of a '\'
+        relativeTarget = relativeTarget.Replace('\\', '/');
```

Comment:
Windows supports both "\\" and "/", but all the `*.expect` files assume that the output is using `/`, hence we normalize using the UNIX directory separator. This is also why we only normalize it at printing time, because it does not bring anything to change it before, and I believe that it's better that this change is made on the last step, for test output consistency only.

Reply:
You could add something like `NormalizedRelativeFilename`.

Reply:
Thanks @keyboardDrummer . I'm sorry for your feeling. As a regular Windows user myself, I'm used to see and use both slashes and backslashes, and I guess that most Windows developers are used to that, so as one of these users you worry about, I can reassure you, it won't negatively impact user experience. Note that this is also used to display informative relative paths in specific logging messages, so it is not an absolute path nor a path used in some API. I myself write paths using slashes most of the time when possible, because when I put backslashes in strings (bash, php, python, JavaScript...), they need to be doubled. This makes scripting quite hard.

The only way to add a global constant to check if we are testing or not would be to change an environment variable, since most tests are actually running whole Dafny processes. But I strongly advise against changing the behavior of an application depending on whether it is tested or not, it makes things even harder to debug (e.g. expectations that are different from a platform to another one).

That said, I think that a good way to implement a change compatible with your thinking would be, in our customized test-aware %diff command, to search and replace the following regex:

    (?<=(?:Wrote textual form of (partial )?target program to |Additional target code written to ).*)\\
    /

Let me know if you still want me to implement the change despite of the above.

Reply:
Alright. If slashes in either direction are the norm in Windows, then we don't need to only output backslashes :-)